src/SignalR/clients/ts/FunctionalTests/web.config

```diff
@@ -7,8 +7,11 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false" startupTimeLimit="3600" requestTimeout="23:00:00">
```

Comment:
I'll revert this

Reply:
Forget?

Reply:
I was making the change as you were typing this lol